src/psyclone/tests/test_files/dynamo0p3/error_import.py

```diff
@@ -0,0 +1 @@
+import non_existent
```

Comment:
Please add a comment here explaining the purpose of this file.

Reply:
`error.py` now `error_syntax.py` never did either...

Reply:
While that might be true, it doesn't mean it's good ;-) Please add an appropriate comment to that file too. (It only needs a single line.)